woocommerce/woocommerce
# Move product editor utils to product editor package

### All Submissions:






### Changes proposed in this Pull Request:

Moves all utils used in the new product editing experience to the `@woocommerce/product-editor` package.  Other components and hooks will come in follow-up PRs.

Closes #36719  .






### How to test the changes in this Pull Request:



1. Navigate to the new product editing experience.
2. Generally smoke test the product editor and make sure it continues to work as expected



### Other information:




### FOR PR REVIEWER ONLY:
